app/src/main/res/layout/activity_member_profile.xml

```diff
@@ -1,159 +1,181 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+            app:layout_constraintTop_toBottomOf="@id/cvImg"
+            tools:text="TextView" />
 
         <TextView
             android:id="@+id/tvUsername"
             android:layout_width="320dp"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="24dp"
             android:layout_marginStart="24dp"
             android:layout_marginTop="16dp"
-            tools:text="TextView"
+            android:layout_marginEnd="24dp"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/tvName" />
+            app:layout_constraintTop_toBottomOf="@+id/tvName"
+            tools:text="TextView" />
 
         <TextView
             android:id="@+id/tvSlackUsername"
             android:layout_width="320dp"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="24dp"
             android:layout_marginStart="24dp"
             android:layout_marginTop="8dp"
-            tools:text="TextView"
+            android:layout_marginEnd="24dp"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/tvUsername" />
+            app:layout_constraintTop_toBottomOf="@+id/tvUsername"
+            tools:text="TextView" />
 
         <TextView
             android:id="@+id/tvAvailableToMentor"
             android:layout_width="320dp"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="24dp"
             android:layout_marginStart="24dp"
             android:layout_marginTop="8dp"
-            tools:text="TextView"
+            android:layout_marginEnd="24dp"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/tvSlackUsername" />
+            app:layout_constraintTop_toBottomOf="@+id/tvSlackUsername"
+            tools:text="TextView" />
 
         <TextView
             android:id="@+id/tvNeedMentoring"
             android:layout_width="320dp"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="24dp"
             android:layout_marginStart="24dp"
             android:layout_marginTop="8dp"
-            tools:text="TextView"
+            android:layout_marginEnd="24dp"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/tvAvailableToMentor" />
+            app:layout_constraintTop_toBottomOf="@+id/tvAvailableToMentor"
+            tools:text="TextView" />
```

Comment:
Please move the string("TextView") to strings.xml with a proper naming convention.

Reply:
Please update all such instances. :)